src/tests/errorMessageGenerator.test.ts

```diff
@@ -0,0 +1,10 @@
+import { scenariosToBeTested } from "./scenariosToBeTested";
+import { errorMessageGenerator } from "../functions/errorMessageGenerator";
```

Comment:
This is superdope.
Maybe I'd suggest instead of having something like `scenarios.map(...).toEqual(...)` to use Jest's [each](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout) built in mechanism.

Reply:
Sure, that looks much cleaner.